patches/minecraft/net/minecraft/block/TrapDoorBlock.java.patch

```diff
@@ -8,7 +8,7 @@
 +   public boolean isLadder(BlockState state, net.minecraft.world.IWorldReader world, BlockPos pos, net.minecraft.entity.LivingEntity entity) {
 +      if (state.func_177229_b(field_176283_b)) {
 +         BlockState down = world.func_180495_p(pos.func_177977_b());
-+         if (down.func_177230_c() == net.minecraft.block.Blocks.field_150468_ap)
++         if (down.func_177230_c() instanceof net.minecraft.block.LadderBlock)
```

Comment:
This by itself isn't bad. Would be nice if we could get a more detailed check using isLadder, but we don't have a directional argument for it...

Reply:
Hi lex, yes, isLadder() was my first check, this is the minimal alternative for a vanilla patch.